tests/test_pwm.py

```
@@ -0,0 +1,168 @@
+        flame_active=flame_active,
+        central_heating=True,
+        hot_water_active=hot_water_active,
+        modulation_reliable=True,
+        flame_on_since=flame_on_since,
+        flame_off_since=None,
+        setpoint=setpoint,
+        flow_temperature=flow_temperature,
+        return_temperature=40.0,
+        max_modulation_level=None,
+        relative_modulation_level=None,
+    )
```

Pylint (4.0.4)

[error] Unexpected keyword argument 'modulation_reliable' in constructor call

(E1123)